Core/src/game/functions/ints/count/sitesPlatformBelow/CountSitesPlatformBelow.java

```diff
@@ -0,0 +1,180 @@
+package game.functions.ints.count.sitesPlatformBelow;
+
+import java.util.BitSet;
+import java.util.List;
+
+import annotations.Hide;
+import annotations.Opt;
+import game.Game;
+import game.functions.ints.BaseIntFunction;
+import game.functions.ints.IntFunction;
+import game.functions.ints.iterator.To;
+import game.types.board.SiteType;
+import game.types.play.RoleType;
+import game.util.directions.AbsoluteDirection;
+import game.util.graph.Step;
+import main.Constants;
+import other.context.Context;
+import other.context.EvalContextData;
+import other.state.container.ContainerState;
+import other.topology.Topology;
+
+/**
+ * Returns the number of groups.
+ * 
+ * @author Eric.Piette & Cedric.Antoine
+ */
+@Hide
+public final class CountSitesPlatformBelow extends BaseIntFunction
+{
+	private static final long serialVersionUID = 1L;
+
+	//-------------------------------------------------------------------------
+
+	/** The graph element type. */
+	private SiteType type;
```

Comment:
this variable seems to be unused (except for by some functions which shouldn't be using it anymore)